xml/obs_best_practice_notifications.xml

```diff
@@ -0,0 +1,288 @@
+<!DOCTYPE sect1
+[
+   <!ENTITY % entities SYSTEM "entity-decl.ent">
+   %entities;
+]>
+
+<sect1 version="5.1" xml:id="notifications_how_to"
+ xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" >
+ <title>Notifications</title>
+ <para>
+  The "Notifications" page is the place that keeps you up to date with your
+  day-to-day OBS work.
+ </para>
+ <para>
+  You can configure them in detail to receive just what is from your interest.
+  Not only that, with the web UI interface, you can easily filter your
+  notifications and mark them as read, improving your OBS workflow experience.
+ </para>
+
+ <sect2 xml:id="cha.obs.best-practices.webuiusage.notifications_configuration">
+  <title>Notifications Configuration</title>
+  <para>
+   Click on the "Change Your Notifications" link to define what notifications
```

Comment:
```suggestion
   Click on the "Change Your Notifications" link to define which notifications
```

Reply:
Well, `what` is fine in this case I guess

Reply:
Both `what` and `which` works, I guess. Also applied.